### PR TITLE
SW.js fix for Netlify

### PR DESCRIPTION
**Pull request by zouhir**
_redirects file is breaking SW.js as it is config file for netlify and trying to HTTP GET it will resolve to 404

this will execlude it from offline plugin.

### CODE REVIEW

File: webpack.config.babel.js [hunk 1]
```diff
@@ -135,6 +135,7 @@ module.exports = {
 		new OfflinePlugin({
 			relativePaths: false,
 			AppCache: false,
+      excludes: ['_redirects'],
```
Comment (NekR) [inline webpack.config.babel.js]: Wrong indentation I guess 😝

Comment (zouhir) [inline webpack.config.babel.js]: Oops! Looks fine on my machine 😂

File: webpack.config.babel.js [hunk 2]
```diff
@@ -134,7 +134,7 @@ module.exports = {
 		new OfflinePlugin({
 			relativePaths: false,
 			AppCache: false,
-			excludes: ['_redirects']
```
Comment (NekR) [inline webpack.config.babel.js]: Lol, you keep ruining `offline-plugin` config :D (just kidding)

**Comment by developit** [comment]
I'll trust you guys on this one :P

**Comment by NekR** [comment]
Risky person, you live on the edge!

**Pull request merged**